packages/valory/skills/inbox_abci/handlers.py

```diff
@@ -272,7 +275,7 @@ def put(self, request: Dict) -> None:
     def add_response(self, response: Dict, db: Optional[str] = None) -> None:
         """Add response to processed list."""
         self._processed.append(response)
-        with open(db or "/logs/db.json", "w") as file:
+        with open(db or self._db, "w") as file:
             json.dump(self._processed, file)
```

Comment:
That's a bit confusing. Why allow passing a `db` other than the one that exists in the object itself?